pymtl3/dsl/NamedObject.py

```diff
@@ -197,16 +307,16 @@ def set_parameter( s, string, value ):
   def _elaborate_construct( s ):
 
     if s._dsl.constructed:
-      print("Don't elaborate the same model twice. \
-             Use APIs to mutate the model.")
+      warnings.warn( "Don't elaborate the same model twice. "
```

Comment:
Interesting. Peitian already mentioned this warning to me. We can discuss whether we want to have warnings throughout this project later. It's fine for now,